src/components/Organisms/AssetTile/AssetTile.story.tsx

```diff
@@ -74,15 +45,23 @@ storiesOf('Design System/Organisms/AssetTile', module)
 
     const withIcon = boolean('with icon', false)
 
+    const selectedFeatures = (options('Features', featureOptions, Object.values(featureOptions), {
+      display: 'check',
+    }) as OptionsTypeKnobValue) as string[]
```

Comment:
Not sure we. need to cast the types here as it's only a story
```suggestion
    }))
```